src/js/index.js

```diff
@@ -30,13 +30,15 @@ import { UserRegistration } from "./components/account/user-registration";
 import { UserSettings } from "./components/account/user-settings";
 import { VerifyUserRegistration } from "./components/account/user-registration-verify";
 
-// Call as early as possible to maximise chance of registering reinstallation code
-tryRegisterSW();
-
 const CONTEXT = `${process.env.CONTEXT}`;
 const SENTRY_DSN = `${process.env.SENTRY_DSN}`;
 const SENTRY_PERF_SAMPLE_RATE = `${process.env.SENTRY_PERF_SAMPLE_RATE}`;
 
+if (CONTEXT === "production") {
+  // Call as early as possible to maximise chance of registering reinstallation code
+  tryRegisterSW();
+}
```

Comment:
I don't think this is needed if we exclude the service worker tag in index.html

Reply:
oh wait it still is needed. hm. I worry then about wanting to dev on service worker features but having to set node env to production

Reply:
Perhaps we can wrap it in a `try/catch` so that it at least does not complete break the app? Perhaps there is something else we can check for that is not `env` related. I will have a look.

Reply:
try/catch for control flow should be avoided. And CONTEXT isn't appropriate for this.

I think NODE_ENV==production or checking if workbox is defined are decent options

Reply:
👍  As discussed we will switch to `NODE_ENV`